### PR TITLE
Remove deprecated classes props

### PR DESCRIPTION
**Pull request by aljones15**
just update so we are not using legacy css classes.

### CODE REVIEW

**Comment by aljones15** [comment]
tested in Chrome, Safari, and Firefox.

Found an issue in Chrome where it does not resize when chrome dev tools are set to a phone's resolution. On the other hand Firefox's responsive mobile views are fine. After a reload Chrome dev tools worked ok and resized in all formats.

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Vue.js create organization wizard component for Bedrock Web apps",
   "main": "index.js",
   "dependencies": {
-    "bedrock-vue-address-form": "digitalbazaar/bedrock-vue-address-form#quasar-v1",
+    "bedrock-vue-address-form": "github:digitalbazaar/bedrock-vue-address-form#quasar-v1",
```
Comment (gannan08) [inline package.json]: Is this change needed?

Comment (gannan08) [inline package.json]: Is this change needed?

**Review by gannan08 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (aljones15) [reply package.json]: no, but keeps it consistent with other package.json files and is not necessary.

Comment (aljones15) [reply package.json]: @gannan08 let me know if this is good to go.

**Comment by gannan08** [comment]
@JoshDunnDev Have you ran this to ensure everything looked good on your end?

**Comment by gannan08** [comment]
@aljones15 This needs a rebase.

**Comment by aljones15** [comment]
@gannan08 rebased with latest from upgrade branch.

**Comment by JoshDunnDev** [comment]
There is a weird issue going on when United States is selected. Not sure exactly what is causing this but the height gets expanded. Then once a State is selected, it reverts back to the correct height.

Screen Shot 2019-04-10 at 3 49 03 PM

Also if we are going to have a filter for Country, we should also have a similar functionality to filter by State as well.

**Comment by gannan08** [comment]
> There is a weird issue going on when United States is selected. Not sure exactly what is causing this but the height gets expanded:
> 
> Screen Shot 2019-04-10 at 3 49 03 PM
> 
> Also if we are going to have a filter for Country, we should also have a similar functionality to filter by State as well.

I'd assume this issue is in the address form. Could be due to the auto-complete feature that was developed. Any ideas on why this might be happening @aljones15 ?

**Comment by aljones15** [comment]
@gannan08 @JoshDunnDev the filter thing actually inserts an input with display: none in the form. anyways looking at this now both in address form and in this module.

the layout of bedrock-vue-address-form also changes when you choose a country (I only tested U.S.A. so far)

**Comment by aljones15** [comment]
weird it is just the USA strange.

@JoshDunnDev @gannan08 this branch seems to fix the issue:

https://github.com/digitalbazaar/bedrock-vue-address-form/tree/toggle-display-none-on-select

longer countries seem to be the issue. if the country is really long it can break the form. the display: none definitely solved the extra hump issue, but 

Screen Shot 2019-04-10 at 5 18 53 PM

even with display: none some countries length does still cause the inputs to be out of alignment. But I think it looks ok in this case.

**Comment by gannan08** [comment]
@JoshDunnDev @aljones15 Going to merge this down as the address form is independent from this package and can be fixed with a patch release.

**Review by gannan08 — APPROVED** [review]
